packages/grid/x-data-grid/src/DataGrid/DataGrid.tsx

```diff
@@ -89,6 +89,24 @@ DataGridRaw.propTypes = {
    * @default 3
    */
   columnBuffer: PropTypes.number,
+  columnGroupingModel: PropTypes.arrayOf(
```

Comment:
Adding proptypes for this prop is not useful. You can add `columnGroupingModel` to https://github.com/mui/mui-x/blob/01aa9c57e7e622ca7a8caa0adbab75f7779b1c90/docs/scripts/generateProptypes.ts#L17